parts/k8s/kuberneteswindowsfunctions.ps1

```diff
@@ -240,24 +240,24 @@ function Write-KubeClusterConfig {
     }
 
     $Global:ClusterConfiguration | Add-Member -MemberType NoteProperty -Name Csi -Value @{
-        EnableProxy = "true"
+        EnableProxy = $global:EnableCsiProxy

```

Comment:
This is the bugfix, everything else is just whitespace cleanup

Reply:
Oh boy, thanks for fixing!